examples/user_guide/03-Applying_Customization.ipynb

```diff
@@ -271,11 +266,11 @@
     "(curve + spikes).cols(1)\n",
     "```\n",
     "\n",
-    "The recommended version declares the `Layout` *before* applying all the options together:\n",
+    "Code like that is very difficult to read because it mixes declarations of the data and its dimensions with details about how to present it.  The recommended version declares the `Layout`, then separately applies all the options in a group where it's clear that they are just hints for the visualization:\n",
```

Comment:
```suggestion
     "Code like that is very difficult to read because it mixes declarations of the data and its dimensions with details about how to present it.  The recommended version declares the `Layout`, then separately applies all the options together where it's clear that they are just hints for the visualization:\n",
     
```

The word 'group' is very overloaded!